cli/compose/loader/full-struct_test.go

```diff
@@ -613,9 +610,6 @@ services:
       BAZ: baz_from_service_def
       FOO: foo_from_env_file
       QUX: qux_from_environment
-    env_file:
```

Comment:
Still pinching my head a bit about this change, and wondering (assuming the `TestMarshallConfig` test shows that we can marshal the config to YAML) if we need a separate test to convert a Service _with_ `env_file` back to YAML.

I agree that this whole test is a bit weird (as in: it's not really symmetrical with the YAML file we _read_)

Reply:
The whole `env_file` resolution is not symmetrical anyways. 
I'm even thinking about another PR to pass an option to avoid it, and by that, conserve the symmetry.